tsconfig.json

```diff
@@ -15,5 +15,8 @@
     "moduleResolution": "node",
     "allowSyntheticDefaultImports": true,
     "noEmit": true
-  }
+  },
+  "include": [
+    "src/**/*"
```

Comment:
Curious about what led to this change?  I think the default is `**/*`.  Was there an error or warning or something?

Reply:
@greinard The images were showing up in VS Code Problems tab as not found despite them showing properly in storybook

Reply:
Which images?  I'm just surprised that `src/**/*` works but the default of `**/*` did not.  Maybe that documentation is incorrect.  🤷‍♂️ 

Are you referring to some image file imports in the stories?  Or in other components?

Reply:
Check out the EhrNewsFeedView.tsx as an example. When that file is open, there a problem item for each imported image.

Reply:
Thanks.  I'll take a look.  👍